core/block/import/converter/collection.go

```diff
@@ -16,6 +19,8 @@ import (
 	"github.com/anyproto/anytype-heart/util/pbtypes"
 )
 
+const importDateLayout = "02.01.2006 15:04:05"
```

Comment:
better to be 2 Jan 2006, because US folks swap month/dat like and it will be ambiguous 01.02.2006

Reply:
so maybe use format from locale?

Reply:
Well we have different languages in Anytype. So maybe `2 Jan 2006` is not suitable for non-english users

Reply:
we can use RFC3339, in this case even the sort by name will be consistent :-)

Reply:
I also support RFC3339 for the same reasons

Reply:
IMHO, RFC3339 looks not really user-friendly) But ok

Reply:
@requilence please check out